tests/cbor_decode/test5_strange/src/main.c

```diff
@@ -340,7 +347,7 @@ void test_map(void)
 	Map_t map;
 
 	zassert_true(cbor_decode_Map(payload_map1, sizeof(payload_map1),
-			&map, true), NULL);
+			&map, NULL), NULL);
```

Comment:
do we loose some validation when passing NULL here instead of the old "true"? If so we should verify once (or more) that the behavior is expected when we pass NULL (ie that all of the input is parsed).

Reply:
Yep, I did add it to some of the tests, just not all the places that were `true` before.